twingate/resource_resource.go

```diff
@@ -322,6 +322,12 @@ func resourceResourceRead(ctx context.Context, resourceData *schema.ResourceData
 		return diag.FromErr(err)
 	}
 
+	// fix active resource state on terraform apply
+	_ = client.updateResourceActiveState(ctx, &Resource{
```

Comment:
Not sure I understand this fix.
Whenever you read a resource you also issue a mutation call to set it to be active? 
Why not check if its already active first?

Reply:
thank you, updated it